src/bundle/ui-dev/src/modules/multi-file-upload/components/upload-list/upload.item.component.js

```diff
@@ -290,196 +201,146 @@ export default class UploadItemComponent extends Component {
 
-        return <Icon customPath={contentTypeIconUrl} extraClasses="ibexa-icon--small-medium" />;
+        return (
+            <>
+                {isStandaloneMode ? (
+                    <Icon name={iconName} extraClasses="ibexa-icon--small" defaultIconName="file" />
+                ) : (
+                    <Icon customPath={contentTypeIconUrl} extraClasses="ibexa-icon--small" />
+                )}
+            </>
+        );
     }
 
-    /**
-     * Renders a progress bar
-     *
-     * @method renderProgressBar
-     * @memberof UploadItemComponent
-     * @returns {null|Element}
-     */
     renderProgressBar() {
-        const { uploaded, aborted, progress, totalSize, uploadedSize, disallowed } = this.state;
+        const { uploaded, aborted, progress, totalSize, uploadedSize, failed } = this.state;
 
-        if (this.props.isUploaded || uploaded || aborted || disallowed) {
+        if (this.props.isUploaded || uploaded || aborted || failed) {
             return null;
         }
 
         return <ProgressBarComponent progress={progress} uploaded={uploadedSize} total={totalSize} />;
     }
 
-    /**
-     * Renders an error message
-     *
-     * @method renderErrorMessage
-     * @memberof UploadItemComponent
-     * @returns {null|Element}
-     */
-    renderErrorMessage() {
-        const { uploaded, aborted, disallowedType, disallowedSize, failed, uploading, disallowedContentType } = this.state;
-        const isError = !uploaded && !aborted && (disallowedSize || disallowedType || disallowedContentType) && failed && !uploading;
-        const cannotUploadMessage = Translator.trans(
-            /*@Desc("Cannot upload file")*/ 'cannot_upload.message',
-            {},
-            'ibexa_multi_file_upload',
-        );
-        const disallowedTypeMessage = Translator.trans(
-            /*@Desc("File type is not allowed")*/ 'disallowed_type.message',
-            {},
-            'ibexa_multi_file_upload',
-        );
-        const disallowedSizeMessage = Translator.trans(
-            /*@Desc("File size is not allowed")*/ 'disallowed_size.message',
-            {},
-            'ibexa_multi_file_upload',
-        );
-        const disallowedContentTypeMessage = Translator.trans(
-            /*@Desc("You do not have permission to create this Content item")*/ 'disallowed_content_type.message',
-            {},
-            'ibexa_multi_file_upload',
-        );
-        let msg = cannotUploadMessage;
-
-        if (disallowedType) {
-            msg = disallowedTypeMessage;
-        }
+    renderErrorInfo() {
+        const { failed, errorMsgs } = this.state;
 
-        if (disallowedSize) {
-            msg = disallowedSizeMessage;
+        if (!failed) {
+            return null;
         }
 
-        if (disallowedContentType) {
-            msg = disallowedContentTypeMessage;
-        }
+        const Translator = getTranslator();
+        const isMultipleErros = errorMsgs.length > 1;
```

Comment:
```suggestion
        const isMultipleErrors = errorMsgs.length > 1;
```
;) or maybe `hasMultipleErrors`?